forch/forchestrator.py

```diff
@@ -739,6 +739,9 @@ def _get_controller_state(self):
             active_state = self._active_state
             if active_state == State.initializing:
                 return State.initializing, 'Initializing'
+            if active_state == State.inactive:
```

Comment:
else if

Reply:
Done

Reply:
Actually this else if not needed, right? Indeed, there seems to be a lint error about this:
```
forch/forchestrator.py:740: [R1705(no-else-return), Forchestrator._get_controller_state] Unnecessary "elif" after "return"
```